benchmarks/mempool.py

```diff
@@ -148,7 +148,7 @@ async def add_spend_bundles(spend_bundles: List[SpendBundle]) -> None:
             for tx in spend_bundles:
                 npc = await mempool.pre_validate_spendbundle(tx, None, tx.name())
                 assert npc is not None
-                _, status, error = await mempool.add_spend_bundle(tx, npc, tx.name())
+                _, status, error = await mempool.add_spend_bundle(tx, npc, tx.name(), rec.height)
```

Comment:
i would use hight instead of rec.height.

unrelated to the pr this whole fake_block_record thing smells, dont think we should pass a block record if it is not needed, the correct thing would probably be pass only the vars you need or a new class that contains only those

Reply:
Thanks - changed